app/controllers/comments_controller.rb

```diff
@@ -0,0 +1,17 @@
+class CommentsController < ApplicationController
+  def create
+    @user = current_user
+    @post = Post.find(params[:post_id])
+    @comment = @post.comments.new(user_id: @user, text: comment_params[:text])
+
+    return unless @comment.save
+
+    redirect_to user_post_path(@post.author, @post)
```

Comment:
- You have done great so far :+1: But there are a few changes you need to make before moving to the next milestone. :+1: 
- [ ] According to the requirement, the form to add a comment should be in the `new_user_post_comment` path, i.e.`/users/:user_id/posts/:post_id/comments/new`. You need to add a `new resource` to comment in your routes file, then create a corresponding `action and view` for that. Under a post. you can have the `add comment` button that redirects to the form for adding comments. :+1: Do well to implement this as required.
- [ ] Also looking at the video below, you can see that even after typing a comment and clicking the add comment button, nothing happens and a new comment is not added. I'll suggest you also fix this while you update your code to follow the requirements I mentioned above. :smile: :wave: :small_airplane: 

- - Video Reference:

https://github.com/Abdulwhbhussain/blog-app/assets/107632780/615843c2-e68c-4a94-a259-10ae32da639f